src/libstd/time/mod.rs

```diff
@@ -95,8 +94,9 @@ impl Instant {
     ///
     /// # Panics
     ///
-    /// This function may panic if the current time is earlier than this instant
-    /// which can happen if an `Instant` is produced synthetically.
+    /// This function may panic if the current time is earlier than this
```

Comment:
Please keep line length consistent with lines elsewhere in the file. From cursory look you reflowed this to 80 characters whereas 100 character line length is used elsewhere.